### PR TITLE
Collision fix2

### PR DESCRIPTION
**Pull request by tetorea**
1) Only the root component of the robot BP is used to handle collisions.
When a robot uses a moving part that can collide with environment objects, we may need to keep the robot reference pose immobile while the skeletal mesh moves.
For such cases, I added 2 RootOffset variables, one for /odom and one for /ue_ros/model_state, that can be modified to correct the reference pose of the robot to compensate for the skeletal mesh local motion.

2) adding a variable to ignore warnings when some links are not available

### CODE REVIEW

**Comment by Tadinu** [comment]
There seem to be files from `Contents`, which have been added to lfs [here](https://github.com/rapyuta-robotics/RapyutaSimulationPlugins/blob/devel/.gitattributes).
Could you try reverting the commit & pulling from lfs then confirm by `git lfs checkout`, which should show no updated asset files.

File: Source/RapyutaSimulationPlugins/Private/Drives/RobotVehicleMovementComponent.cpp [hunk 1]
```diff
@@ -159,11 +159,10 @@ void URobotVehicleMovementComponent::UpdateMovement(float InDeltaTime)
                 // Moves the robot up or down, depending on impact position
                 float minDistance = *Algo::MinElement(contactsDistance);
                 minDistance -= RayOffsetUp;
-
                 minDistance = FMath::Min(minDistance, FallingSpeed * InDeltaTime);
 
                 FVector heightVariation = {0., 0., -minDistance};
-                PawnOwner->AddActorWorldOffset(heightVariation, true, &hit, ETeleportType::None);
+                PawnOwner->AddActorWorldOffset(heightVariation, false, &hit, ETeleportType::None);
```
Comment (yuokamoto) [inline Source/RapyutaSimulationPlugins/Private/Drives/RobotVehicleMovementComponent.cpp:164]: any reason to make it false?

File: Source/RapyutaSimulationPlugins/Private/Drives/RobotVehicleMovementComponent.cpp [hunk 2]
```diff
@@ -324,16 +328,15 @@ void URobotVehicleMovementComponent::InitMovementComponent()
     if (bIsFloorHit)
     {
         MinDistanceToFloor = hitResult.Distance;
-        UE_LOG(LogTemp,
-               Warning,
-               TEXT("URobotVehicleMovementComponent::InitMovementComponent - Min Distance To Floor = %f"),
-               MinDistanceToFloor);
     }
+    UE_LOG(LogTemp,
```
Comment (yuokamoto) [inline Source/RapyutaSimulationPlugins/Private/Drives/RobotVehicleMovementComponent.cpp]: Does this woring output every tick?

File: Source/RapyutaSimulationPlugins/Private/Drives/RobotVehicleMovementComponent.cpp (hunk 1 above)
Comment (tetorea) [reply Source/RapyutaSimulationPlugins/Private/Drives/RobotVehicleMovementComponent.cpp:164]: Here I sometimes got collisions between floor and robot where robot was visibly few centimeters above the floor, although `show collision` terminal command did not show any collisions.
At this point, collision checks between contact points and floor has already been computed so not sweeping is possible. But of course, there are possibilities that floor concavities collide with the robot body parts far from contact points then. In current project, that does not happen in normal cases.

File: Source/RapyutaSimulationPlugins/Private/Drives/RobotVehicleMovementComponent.cpp (hunk 2 above)
Comment (tetorea) [reply Source/RapyutaSimulationPlugins/Private/Drives/RobotVehicleMovementComponent.cpp]: Only once at the beginning. This function is called in the constructor.

**Comment by tetorea** [comment]
> There seem to be files from `Contents`, which have been added to lfs [here](https://github.com/rapyuta-robotics/RapyutaSimulationPlugins/blob/devel/.gitattributes). Could you try reverting the commit & pulling from lfs then confirm by `git lfs checkout`, which should show no updated asset files.

Done, but is there a way to check I've done it correctly?

**Comment by Tadinu** [comment]
Yeah, after pulling from lfs, those real assets will be downloaded from lfs server to your PC, overriding pointers also, then by `git lfs checkout`, it should show no updates. Anw, they have been excluded from changed files now👍🏼

**Review by Tadinu — APPROVED** [review]
Yeah, after pulling from lfs, those real assets will be downloaded from lfs server to your PC, overriding pointers also, then by `git lfs checkout`, it should show no updates. Anw, they have been excluded from changed files now👍🏼
Thanks, could you update `LogTemp` -> `LogRapyutaCore` for consistency?

**Comment by Tadinu** [comment]
Yeah, after pulling from lfs, those real assets will be downloaded from lfs server to your PC, overriding pointers also, then by `git lfs checkout`, it should show no updates. Anw, they have been excluded from changed files now👍🏼
Thanks, could you update `LogTemp` -> `LogRapyutaCore` for consistency?

**Review by yuokamoto — APPROVED** [review]

**Pull request merged**